### PR TITLE
docs: update export tool doc

### LINKED ISSUE #1112: Update docs for feat: export all schemas and data at once in export tool
A document change request is generated from https://github.com/GreptimeTeam/greptimedb/pull/4478

### PR DESCRIPTION
**Pull request by fengjiachun**
## What's Changed in this PR

As the title said

This PR closes #1112 
## Checklist

- [ ] Please confirm that all corresponding versions of the documents have been revised.
- [ ] Please ensure that the content in `summary.yml` matches the current document structure when you changed the document structure.
- [ ] This change requires follow-up update in localized docs.

### CODE REVIEW

File: docs/nightly/en/user-guide/upgrade.md [hunk 1]
```diff
@@ -75,47 +75,51 @@ OPTIONS:
 
 Here explains the meaning of some important options
 
-- `--addr`: The gRPC address of the Frontend node or Standalone process.
+- `--addr`: The server address of the Frontend node or Standalone process.
 - `--output-dir`: The directory to put the exported data. Give a path at your current machine. The exported SQL files will be put in that directory.
-- `--target`: The things to export. `create-table` can export the `CREATE TABLE` clause for each table. `database-data` can export the data of each database alone with `COPY FROM` clause.
+- `-target`: The content to be exported. It is recommended not to fill it in and use the default value directly, which will export all schemas by default, that is, the `CREATE-TABLE` statements. At the same time, it will export data files for each table and `COPY FROM` statements used for re-importing data into a new DB.
```
Comment (nicecui) [inline docs/nightly/en/user-guide/upgrade.md:80]:
```suggestion
- `-target`: Specifies the data to export. The `create-table` option exports the `CREATE TABLE` clause for each table. The `database-data` option exports the data of each database along with the `COPY FROM` clause. By default, all data is exported for both `create-table` and `database-data`. It is recommended not to specify this option that use the default value to export all data.
```

File: docs/nightly/zh/user-guide/upgrade.md [hunk 2]
```diff
@@ -71,13 +71,14 @@ OPTIONS:
...
+
 ```
 
 这里解释一些重要选项的含义：
 
-- `-addr`：Frontend 节点或者 Standalone 进程的 gRPC 地址。
+- `-addr`：Frontend 节点或者 Standalone 进程的 http server 地址。
 - `-output-dir`：要放置导出数据的目录。需要是当前机器上的路径。导出的 SQL 文件将放在该目录中。
-- `-target`：要导出的内容。`create-table` 可以导出每个表的 `CREATE TABLE` 语句。`database-data` 可以导出每个表的数据以及对应 DB 的 `COPY FROM` 语句。
+- `-target`：要导出的内容。建议不填直接使用默认值，默认会导出全部 schema 也就是 `CREATE-TABLE` 语句，同时导出每个表的数据文件和用于数据再次导入新 DB 的 `COPY FROM` 语句。
```
Comment (nicecui) [inline docs/nightly/zh/user-guide/upgrade.md:81]:
```suggestion
- `-target`：指定要导出的数据。`create-table` 选项会导出每个表的 `CREATE TABLE` 子句。`database-data` 选项会导出每个数据库的数据以及对应 DB 的 `COPY FROM` 语句。默认情况下会导出 `create-table` 和 `database-data` 的所有数据，推荐不设定选项直接使用默认值以导出所有数据。
```

**Review by nicecui — COMMENTED** [review]
please update v0.9 if necessary

File: docs/nightly/en/user-guide/upgrade.md [hunk 3]
```diff
@@ -75,47 +77,51 @@ OPTIONS:
 
 Here explains the meaning of some important options
 
-- `--addr`: The gRPC address of the Frontend node or Standalone process.
+- `--addr`: The server address of the Frontend node or Standalone process.
 - `--output-dir`: The directory to put the exported data. Give a path at your current machine. The exported SQL files will be put in that directory.
-- `--target`: The things to export. `create-table` can export the `CREATE TABLE` clause for each table. `database-data` can export the data of each database alone with `COPY FROM` clause.
+- `-target`: Specifies the data to export. The `create-table` option exports the `CREATE TABLE` clause for each table. The `database-data` option exports the data of each database along with the `COPY FROM` clause. By default, all data is exported for both `create-table` and `database-data`. It is recommended not to specify this option that use the default value to export all data.
```
Comment (fengjiachun) [inline docs/nightly/en/user-guide/upgrade.md:82]:
```suggestion
- `-target`: Specifies the data to export. The `schema` option exports the `CREATE TABLE` clause for each table. The `data` option exports the data of each database along with the `COPY FROM` clause. By default, all data is exported for both `schema` and `data`. It is recommended not to specify this option that use the default value to export all data.
```

File: docs/nightly/zh/user-guide/upgrade.md [hunk 4]
```diff
@@ -75,9 +77,9 @@ OPTIONS:
 
 这里解释一些重要选项的含义：
 
-- `-addr`：Frontend 节点或者 Standalone 进程的 gRPC 地址。
+- `-addr`：Frontend 节点或者 Standalone 进程的 http server 地址。
 - `-output-dir`：要放置导出数据的目录。需要是当前机器上的路径。导出的 SQL 文件将放在该目录中。
-- `-target`：要导出的内容。`create-table` 可以导出每个表的 `CREATE TABLE` 语句。`database-data` 可以导出每个表的数据以及对应 DB 的 `COPY FROM` 语句。
+- `-target`：指定要导出的数据。`create-table` 选项会导出每个表的 `CREATE TABLE` 子句。`database-data` 选项会导出每个数据库的数据以及对应 DB 的 `COPY FROM` 语句。默认情况下会导出 `create-table` 和 `database-data` 的所有数据，推荐不设定选项直接使用默认值以导出所有数据。
```
Comment (fengjiachun) [inline docs/nightly/zh/user-guide/upgrade.md:82]:
```suggestion
- `-target`：指定要导出的数据。`schema` 选项会导出每个表的 `CREATE TABLE` 子句。`data` 选项会导出每个数据库的数据以及对应 DB 的 `COPY FROM` 语句。默认情况下会导出 `schema` 和 `data` 的所有数据，推荐不设定选项直接使用默认值以导出所有数据。
```

**Review by nicecui — APPROVED** [review]

**Review by WenyXu — APPROVED** [review]

**Pull request merged**